src/FormParserFactory.php

```diff
@@ -10,11 +10,12 @@ class FormParserFactory
     public static function create(Request $request)
     {
         $headers = $request->getHeaders();
+        $headers = array_change_key_case($headers, CASE_LOWER);
 
-        if (!array_key_exists('Content-Type', $headers)) {
+        if (!array_key_exists('content-type', $headers)) {
         }
 
-        $contentType = strtolower($headers['Content-Type']);
+        $contentType = strtolower($headers['content-type']);
 
```

Comment:
There is no `Content-Length` in `RequestParser` anymore, it moved to `FormUrlencoded` where this was indeed applied: https://github.com/reactphp/http/blob/streaming-multipart/src/Parser/FormUrlencoded.php#L38-L39 :smile: